File: src/main/java/com/wanderers/hotelier_webservices/rest/exception/ExpiredTokenException.java
package com.wanderers.hotelier_webservices.rest.exception;

public class ExpiredTokenException extends RESTException {

    private static final long serialVersionUID = 1L;

    /**
     * Class constructor
     *
     * @param message String value of Exception
     */
    public ExpiredTokenException(String message) {
        super(message);
    }
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/rest/exception/ExpiredTokenException.kt
package com.wanderers.hotelier_webservices.rest.exception

class ExpiredTokenException
/**
 * Class constructor
 *
 * @param message String value of Exception
 */
    (message: String?) : RESTException(message) {
    companion object {
        private const val serialVersionUID = 1L
    }
}